### ISSUE TITLE
Loading 

### ISSUE THREAD

**Issue by Ikiboyno** [michael5r/mmm-hue-lights#17]
I followed exactly the stepts but when MM start it doesnt is just showing loading. The hue bridge is connected. PLease advice as i dont have advance knowledge in coding . Thank you . 
![WhatsApp Image 2020-04-10 at 14 18 44](https://user-images.githubusercontent.com/17007059/78993717-3ec88300-7b36-11ea-9ced-79c6f98fd63d.jpeg)

**Comment by JohnLucaJF** [comment]
So you can GET & POST to get user id ok?

**Comment by cain0z** [comment]
I am having the same issue. But i can do Get and Post

**Comment by Omeliel** [comment]
HI. Just pay attention to the install. Just go slowly and read the steps. I fixed mine by reading it again . 
So from where you download the module follow exactly the steps. It should work .

**Comment by michael5r** [comment]
@Ikiboyno If you're getting a `Loading` message that never stop, it means there's a problem getting the data.

Double check your config settings to make sure you've added the correct `bridgeIp` and `user`.

Another thing you can try is go into the `mmm-hue-lights.js` file, and comment out this piece of code on line 768:

```
if (this.loaded) {

    if (numberOfOldLights !== numberOfLights) {
        // number of lights changed (update dom)
    } else if (numberOfOldGroups !== numberOfGroups) {
        // number of groups changed (update dom)
    } else {
        // compare status of lights
        if ((this.jsonEqual(oldLights,lights)) && (this.jsonEqual(oldGroups,groups))) {
            renderUi = false;
        }
    }

}
```

**Comment by michael5r** [comment]
I've released a new `1.4.0` version of this module that will hopefully help with the refresh issue - it now automatically reschedules an update, no matter whether there's an error or not.

Please download the version and see if that solves your issue.
